package/extensions/page-break/page-break.ts

```diff
@@ -138,4 +122,8 @@ export const PageBreak = Node.create<PageBreakRuleOptions>({
       }),
     ];
   },
+
+  addNodeView() {
+    return ReactNodeViewRenderer(PageBreakNodeView as any);

```

Comment:
Do we necessarily need type any here? If not, please remove it and the eslint disable as well

Reply:
Yes on the first line